Comanche055/P32-P33_P72-P73.agc

```diff
@@ -39,7 +39,7 @@
 # COELLIPTIC SEQUENCE INITIATION (CSI) PROGRAMS (P32 AND P72)
 #
 # MOD NO -1		LOG SECTION -- P32-P35, P72-P75
-# MOD BY WHITE. P	DATE 1JUNE67
```

Comment:
Rejected. Should be `WHITE.P` without spaces ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0649.jpg))